src/API/Management/Users.php

```diff
@@ -1,88 +1,265 @@
 <?php
-
+/**
+ * Users endpoints for the Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
 namespace Auth0\SDK\API\Management;
 
-use Auth0\SDK\API\Header\ContentType;
-
-class Users extends GenericResource {
-
-    public function get($user_id) {
-
-        return $this->apiClient->get()
-            ->users($user_id)
+/**
+ * Class Users.
+ * Handles requests to the Users endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
+class Users extends GenericResource
+{
+    /**
+     * Get a single User by ID.
+     * Requires scopes: read:users, read:user_idp_tokens.
+     *
+     * @param string $user_id - User ID to get.
+     *
+     * @return mixed
+     *
+     * @throws \Exception
+     */
+    public function get($user_id)
+    {
+        return $this->apiClient->method('get')
+            ->addPath('users', $user_id)
             ->call();
     }
 
-    public function update($user_id, $data) {
-        return $this->apiClient->patch()
-            ->users($user_id)
-            ->withHeader(new ContentType('application/json'))
+    /**
+     * Update a User.
+     * Requires scope: update:users.
+     * May require scope: update:users_app_metadata
```

Comment:
ahá!! again, _when_ would this scope be required? Don't say it's implicit on the scope value. You can also rephrase it in a similar way to this:

> Requires scope 'update:users'. If the values being changed include the 'user_metadata' or 'app_metadata' fields, additionally the scope must include 'update:users_app_metadata'.